_episodes/06-agile-in-research.md

```diff
@@ -0,0 +1,138 @@
+'Process-rigidity is the antithesis of agility.'
+- **Emphasize the "why"**: Focus on the "why" for the change to agile instead of
+"what" to do to make the change.
+
+----------------------------------------------------
+### Waterfall project setup challenge
+Scientific projects are often setup in long-term project timelines following a
+Waterfall method.
+#### Solutions:
+- Clear communication is key. Inform your collaborators on your way of working and update them frequently.
+- Actually projects that follow the Waterfall method often do not make the targets set out at the start of the project.
+
+----------------------------------------------------
+### Large-scale projects challenge
+Emergent design in iterative sprints does not always lead to the best large-scale functionality.
+#### Solutions
+-  It makes sense to do a few design sprints to set out the bigger plan before embarking on a project.
+- The [Architectural Runway](https://www.scaledagileframework.com/architectural-runway/)
+approach revolves around more large-scale "intentional architectures".
+
+----------------------------------------------------
+### Team dedication challenge
+Agile is often explained with the idea of a team fully dedicated to a single project.
+In academia team members are often involved in numerous projects in and outside of the team.
+#### Solutions:
+- **Implement flexibly**: Again: You should fit the methodology to the way of working.
+Start from where your team's activities do overlap and where teamwork seems most beneficial.
+- **Keeping track**: Experiment and iterate on ideas to find the best way to keep track of
+the different projects in your team. You can start with a Kanban board and backlog for each project seperately.
+
+----------------------------------------------------
+### Product owner challenge
+For a team running multiple research projects, who acts as product owner?
+#### Solutions:
+- Each project has its own 'project owner' (usually a team member) that takes
+the role of product owner for that project.
+
+------------------------------------------------
+### Distributed expertise challenge
+In Agile methodologies it is often assumed that a task can be picked up by any team member.
+However, in science different team members often have specialized in certain domains or technologies,
+and it makes more sense to assign specific tasks to members with a specific expertise.
+#### Solutions:
+- Not everyone has to be involved in everything!
+- Be explicit and open in each team members expertise and how that fits best to the different projects.
+
+----------------------------------------------------
+### What is value challenge
+The goal of Agile way of working is to maximize value for the customer,
+but what is value in research? And who is the customer?
+#### Solutions:
+- Discuss this in your team! Our proposition:
+    - Our customers: The Scientific Community at Large
```

Comment:
Agreed, although this is most frequently specific scientific communities, non-scientific communities (e.g. research software for researchers in the humanities etc), potentially "customers" like the EU Commission...